### ISSUE TITLE
Configure the pull interval

### ISSUE THREAD

**Issue by allebacco** [scribd/objinsync#10]
Is it possible to configure the interval between synchronizations with the S3 buckets?

From the code, I have seen that it is 5 seconds, but, since I have never used the go language, I don't know how it can be configured using CLI like for the `--once` or `--exclude` arguments.

I would like to be able to write:
```bash
objinsync pull --interval 60 s3://bucket/keyprefix ./localdir
```

**Comment by houqp** [comment]
@allebacco this should be an easy change, the value is currently hardcoded at https://github.com/scribd/objinsync/blob/master/main.go#L101. You should be able to replace that with a duration typed flag https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Duration.

So something like this:

```go
	pullCmd.PersistentFlags().DurationVarP(
		&FlagPullInterval, "interval", "i", time.Second * 5, "Interval between remote storage pulls")
```

**Comment by allebacco** [comment]
@houqp Thanks, your suggestion will be very helpful.

**Comment by houqp** [comment]
fixed in release `2.4.0`.
